products/sql/api.yaml

```diff
@@ -17,7 +17,7 @@ display_name: Cloud SQL
 versions:
   - !ruby/object:Api::Product::Version
     name: ga
-    base_url: https://www.googleapis.com/sql/v1beta4/
+    base_url: https://www.googleapis.com/
```

Comment:
I think we want the value from here https://github.com/terraform-providers/terraform-provider-google/pull/5584/files#diff-d896461fc49170c900b393a61f754b89R79 with `v1beta4` still appended. The format we use in MM is a little made up.

Reply:
I've made this change as you've suggested.  It still seems a little odd in a few of the downstreams since `sql` is still taken out of the url.  Will that be something that the downstreams will have to address somehow?

Reply:
I think we want the first part to be `https://sqladmin.googleapis.com/` instead of `https://www.googleapis.com/` , that's what appears in the linked code.

Reply:
oh, sorry, yes, i saw that - it just didn't register in my thought process. thanks!